backend/supplies/tests.py

```diff
@@ -144,3 +146,38 @@ def test_editing_supplies_unauthenticated(self):
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
         # data should not change
         self.assertEqual(Supply.objects.get(id=id).state, oldstate)
+
+
+class SearchSupplyTests(TestCase):
```

Comment:
1. test_searching should be divided into three separate unit tests:
- test_search_multiple_results
- test_search_single_result
- test_search_no_results
2. Please include SupplyListView's order-filtering endpoint tests.
3. Both of test classes (SupplyTests, SearchSupplyTest) in supplies/tests.py should inherit APITestCase (being DRF exclusive) instead of TestCase (being regular Django). Also their names could be more straight forward (SupplyTests should tell what exactly these tests include, because as of now it's just general).

Other than that looks fine to me and can be merged once points above are resolved.

Reply:
Fixed these issues.